mmv1/products/netapp/volume.yaml

```diff
@@ -245,6 +264,32 @@ properties:
     description: |
       Reports the resource name of the Active Directory policy being used. Inherited from storage pool.
     output: true
+  - !ruby/object:Api::Type::NestedObject
+    name: 'restoreParameters'
+    description: |-
+      Used to create this volume from a snapshot (= cloning) or an backup.
+    immutable: true
+    # This parameter is only used at CREATE. READs will omit it.
+    ignore_read: true
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'sourceSnapshot'
+        description: |-
+          Full name of the snapshot to use for creating this volume.
+          `source_snapshot` and `source_backup` cannot be used simultaneously.
+          Format: `projects/{{project}}/locations/{{location}}/volumes/{{volume}}/snapshots/{{snapshot}}`.
+        conflicts:
```

Comment:
Do you think if it makes sense to set [`exactly_one_of`](https://googlecloudplatform.github.io/magic-modules/develop/field-reference/#exactly_one_of) for these two fields instead of `conflicts` - it will prevent users from specifying empty block `restore_parameters {}`

Reply:
That's a great idea. Will give it a try.

Reply:
Done